tests/system/common.py

```diff
@@ -721,7 +721,7 @@ def __marathon_leadership_changed_in_mesosDNS(original_leader):
     assert original_leader != current_leader
 
 
-@retrying.retry(wait_fixed=1000, stop_max_attempt_number=10, retry_on_exception=ignore_exception)
+@retrying.retry(wait_fixed=1000, stop_max_attempt_number=60, retry_on_exception=ignore_exception)
```

Comment:
Should we use exponential backoff instead?

See the [examples](https://github.com/rholder/retrying#examples)

```python
@retry(wait_exponential_multiplier=1000, wait_exponential_max=10000)
```